client/pages/add-course.tsx

```diff
@@ -22,19 +25,17 @@ const Home: NextPage = () => {
                 <Text>
                 This module covers knowledge and application skills in understanding the importance of service innovation and the methods and opportunities that are available for a service staff to generate ideas that contribute to service innovation.
                 </Text>
-                <Spacer/>               
-                <Spacer/>
-                <Link href="/addCourse">
-                <Button type="secondary">Add Course</Button>
-                </Link>
+                <Spacer/>    
+                <Toast></Toast>           
```

Comment:
```suggestion
                <Toast text={"Successfully enrolled."} />       
```

Reply:
We can close the component in a more react-y way with `<Toast />` instead of `<Toast></Toast>`. The addition of prop `text` follows the changes in the above comment.